src/cpu/aarch64/jit_uni_reorder.cpp

```diff
@@ -320,8 +318,11 @@ struct jit_uni_reorder_kernel_f32_t : public kernel_t, public jit_generator {
         /* 4th turn */
         for (uint32_t i = 0; i < unroll / 2; i++) {
             ZRegB z {unroll / 2 + i};
+            /* Move bit 128-255 to 0-127. */
             ext(z, z, 16);
-            ext(z_tmp_vec[unroll / 2 + i].b, z_tmp_vec[unroll / 2 + i].b, 48);
+            /* Move bit 0-127 to 128-255. */
+            ext(z_tmp_vec[unroll / 2 + i].b, z_tmp_vec[unroll / 2 + i].b,
```

Comment:
```suggestion
            ZRegB z_tmp = z_tmp_vec[unroll / 2 + i].b;
            ext(z_tmp, z_tmp,
```

Reply:
Thank you for the suggestion. I will apply it.